README.md

```diff
@@ -54,8 +70,30 @@ public class SystemLivenessCheck implements HealthCheck {
 ```
 </details>
 
+Notice that the `src/main/liberty/config/configDropins/overrides/generated-features.xml` file was updated and now contains the `mpHealth-3.0` feature.
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<server>
+    <!--This file was generated by the Liberty Maven Plugin and will be overwritten on subsequent runs of the liberty:generate-features goal.
+It is recommended that you do not edit this file and that you commit this file to your version control.-->
+    <featureManager>
+        <!--The following features were generated based on API usage detected in your application-->
+        <feature>cdi-2.0</feature>
+        <feature>mpHealth-3.0</feature>
+        <feature>mpMetrics-3.0</feature>
+        <feature>jaxrs-2.1</feature>
+    </featureManager>
+</server>
+```
+
+4. Delete the src/main/java/io/openliberty/sample/system/SystemLivenessCheck.java class. Notice that the `mpHealth-3.0` feature still exists in `src/main/liberty/config/configDropins/overrides/generated-features.xml`.
```

Comment:
Should we explain why? E.g. "Dev mode does not update this file when a class is deleted." Or "Dev mode retains all the features in case other class files need them. To remove a feature dev mode would need to re-scan all the class files."